db/seeds.rb

```diff
@@ -32,5 +44,13 @@ def make_email(data)
       }
     )
   )
+
   user.save
+
+  (0..Random.new.rand(5..20)).each do |_i|
```

Comment:
There is a simpler way to do that.
```suggestion
  rand(5..20).times do
```
Btw why do you create a random number of articles?

Reply:
no reason